### PR TITLE
set up redux on login page

### PR DESCRIPTION
**Pull request by kspecie**

### CODE REVIEW

File: client/components/LoginForm.jsx [hunk 1]
```diff
@@ -9,20 +11,29 @@ const LoginForm= () => {
...
       setInputUsernameValue(event.target.value);
     }
     const handlePasswordChange = (event) => {
       setInputPasswordValue(event.target.value);
     }
 
-    const handleLoginSubmit = () => {
-      alert(`Username: ${inputUsernameValue} Password: ${inputPasswordValue}`)
-    }
+    // const handleLoginSubmit = () => {
```
Comment (ryangilbert) [inline client/components/LoginForm.jsx:23]: can this be removed?

Comment (kspecie) [reply client/components/LoginForm.jsx:23]: yes

**Review by ryangilbert — APPROVED** [review]
left one comment, not required, but lgtm!

File: client/actions/loginAction.js
Comment (ryangilbert) [inline client/actions/loginAction.js:1]: empty file, can this be removed?

**Pull request merged**